src/pyOpenMS/pyopenms/_dataframes.py

```
@@ -79,3 +114,63 @@ def update_scores_from_df(peps: _PeptideIdentificationList, df: _pd.DataFrame, m
+                elif dv.valueType() == _DataValue.EMPTY_VALUE:
+                    continue
+                else:
+                    value = str(dv)
+                    dtype = "object"
+            else:
+                value = str(dv)
+                dtype = "object"
+
+            df[col_name] = _np.full(df.shape[0], value, dtype=dtype)
+
+        except Exception:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)